.devcontainer/devcontainer.json

```diff
@@ -1,26 +1,25 @@
 {
     "name": "Azure Developer CLI",
-    "image": "mcr.microsoft.com/devcontainers/javascript-node:18-bookworm",
+    "image": "mcr.microsoft.com/devcontainers/typescript-node:dev-18-bookworm",
     "features": {
-        "ghcr.io/azure/azure-dev/azd:latest": {},
-        "ghcr.io/devcontainers/features/azure-cli:latest": {},
         "ghcr.io/devcontainers/features/docker-in-docker:latest": {},
-        "ghcr.io/devcontainers/features/github-cli:latest": {},
-        "ghcr.io/devcontainers/features/powershell:latest": {}
+        "ghcr.io/azure/azure-dev/azd:latest": {},
+        "ghcr.io/devcontainers-contrib/features/npm-package:latest": {},
+        "ghcr.io/devcontainers/features/azure-cli:latest": {}
     },
     "customizations": {
         "vscode": {
             "extensions": [
-                "dbaeumer.vscode-eslint",
-                "esbenp.prettier-vscode",
                 "GitHub.vscode-github-actions",
                 "ms-azuretools.azure-dev",
+                "ms-azuretools.vscode-azurefunctions",
                 "ms-azuretools.vscode-bicep",
                 "ms-azuretools.vscode-docker",
+                "ms-vscode.vscode-node-azure-pack",
+                "dbaeumer.vscode-eslint",
+                "esbenp.prettier-vscode",
                 "ms-vscode.azurecli",
-                "ms-vscode.js-debug",
-                "ms-vscode.powershell",
```

Comment:
why remove powershell?

Reply:
@dotnet-7 It used to be there in my original `devcontainer.json` settings. Please revert.